aws/resource_aws_batch_compute_environment.go

```diff
@@ -0,0 +1,496 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/batch"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+const (
+	MANAGED   = "MANAGED"
```

Comment:
Sorry if my comment was misleading, I meant to prefer the use of constants from the SDK, as exposed [here](https://github.com/terraform-providers/terraform-provider-aws/blob/master/vendor/github.com/aws/aws-sdk-go/service/batch/api.go#L4586)